web/client/components/data/identify/__tests__/DefaultViewer-test.jsx

```diff
@@ -53,6 +53,11 @@ describe('DefaultViewer', () => {
             layerMetadata: {
                 title: 'a'
             }
+        }, {
```

Comment:
these tests changes are not needed anymore, for the whole file, you are not modifying this anymore

Reply:
true, Im going to revert all my changes to this file